modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/DefaultRenderer.js

```diff
@@ -35,6 +49,19 @@ function DefaultRenderer({value}) {
 		return <TooltipTextRenderer value={value} />;
 	}
 	else if (value.label) {
+		if (options.truncate) {
+			return (
+				<span
+					className={classNames(
+						'default-renderer__text-truncate',
+						'text-truncate'
+					)}
+				>
+					{value.label}
+				</span>
+			);
+		}
+
```

Comment:
We should avoid code duplication. One way is to add something like:

```js
const Wrapper = ({options}) => {
	return options.truncate ? (
		<span
			className={classNames(
				'default-renderer__text-truncate',
				'text-truncate'
			)}
		>
			{children}
		</span>
	) : (
		{children}
	);
};
```
... and call it from two places.

There are other, maybe better ways.

Reply:
Please fix this as well @danvb112